static/app/views/explore/charts/index.tsx

```diff
@@ -1,5 +1,134 @@
-interface ExploreChartsProps {}
+import styled from '@emotion/styled';
 
-export function ExploreCharts({}: ExploreChartsProps) {
-  return <div>TODO: visualize charts</div>;
+import {getInterval} from 'sentry/components/charts/utils';
+import {CompactSelect} from 'sentry/components/compactSelect';
+import {DEFAULT_RELATIVE_PERIODS} from 'sentry/constants';
+import {CHART_PALETTE} from 'sentry/constants/chartPalette';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {tooltipFormatter} from 'sentry/utils/discover/charts';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import Chart, {ChartType} from 'sentry/views/insights/common/components/chart';
+import ChartPanel from 'sentry/views/insights/common/components/chartPanel';
+import {useSpanIndexedSeries} from 'sentry/views/insights/common/queries/useDiscoverSeries';
+import {CHART_HEIGHT} from 'sentry/views/insights/database/settings';
+import {useMetricsIntervalParam} from 'sentry/views/metrics/utils/useMetricsIntervalParam';
```

Comment:
We may want to write our own version of this as this implementation seems to make a few assumptions around the mri/widgets on the metrics page that do not apply here.

Reply:
Hey yup, I'll get it done @Zylphrex